test/supports/class_hierarchy.py

```diff
@@ -0,0 +1,20 @@
+import modal
+
+app = modal.App("class-hierarchy")
+
+
+class Base:
+    @modal.method()
+    def defined_on_base(self):
+        print("base")
+
+    @modal.method()
+    def overridden_on_wrapped(self):
+        raise NotImplementedError()
+
```

Comment:
Was hoping this would also fix the bug where `modal run my_app.py::Wrapped.overridden_on_wrapped` executes the base version of the method, but apparently that is a separate defect.

Reply:
Resolved here: https://github.com/modal-labs/modal-client/pull/2590